tests/cases/standalone/common/truncate/truncate.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE monitor (host STRING, ts TIMESTAMP, cpu DOUBLE DEFAULT 0, memory DOUBLE, TIME INDEX (ts), PRIMARY KEY(host));
+
+INSERT INTO monitor(ts, host, cpu, memory) VALUES
+(1695217652000, 'host1', 66.6, 1024),
+(1695217652000, 'host2', 66.6, 1024),
+(1695217652000, 'host3', 66.6, 1024),
+(1695217654000, 'host1', 77.7, 2048),
+(1695217654000, 'host2', 77.7, 2048),
+(1695217654000, 'host3', 77.7, 2048),
+(1695217656000, 'host1', 88.8, 4096),
+(1695217656000, 'host2', 88.8, 4096),
+(1695217656000, 'host3', 88.8, 4096);
+
+SELECT ts, host, cpu, memory FROM monitor ORDER BY ts;
+
+TRUNCATE monitor;
```

Comment:
What if the user truncates the table many times? And should we add tests like:

```
TRUNCATE monitor;
INSERT INTO ...
TRUNCATE monitor;
```

Reply:
Thank you for your suggestion.